utils/asn_update.sh

```diff
@@ -12,104 +10,137 @@ function processing_list() {
 }
 
 function create_list() {
+	RETVAL=0
 	LIST=/tmp/list
 
 	for i in "${@:3}"; do
-		./get_routes_by_asn.sh "$i" >> $LIST
+		if ! ./get_routes_by_asn.sh "$i" >> $LIST; then
+			echo "Could not fetch route for ${i} (${1})"
+			RETVAL=$(( RETVAL + $? ))
+		fi
```

Comment:
You removed  all the ASs without any prefixes.
Do we have any guarantees that these ASs don't start announcing some prefixes in the next future? If so, this change make sense. Otherwise I think it is more "future proof" to keep them and keep polling them

Reply:
The goal was to let the script fail if e.g. ASN update does not work anymore. I will re-add them and look for another solution.